New Text Document.txt

```diff
@@ -0,0 +1,8 @@
+1. remove background pic 
```

Comment:
probably want to remove this from the repo - we don't want public record of our task notes here :)